### PR TITLE
[SB] Update basil error present status

### PR DESCRIPTION
**Pull request by ssettipalli**
Update basil with error_present status
and fix studyBuilder NPE

### CODE REVIEW

File: study-builder/studies/basil/i18n/es.conf [hunk 1]
```diff
@@ -37,7 +37,8 @@
     "summary_created": "[es] Completing the Loved One Survey will tell us about your loved one's experiences eating Basil",
     "summary_in_progress": "[es] Submitting the Loved One Survey will tell us about your loved one's experiences eating Basil",
     "complete": "[es] All set - thank you for letting us know about your loved one's basil eating experience",
-    "summary_error_present" : "[es] Error Present. Please fix errors and resumit"
+    "summary_error_present" : "[es] Error Present. Please fix errors and resumit",
```
Comment (Trimax) [inline study-builder/studies/basil/i18n/es.conf]: resumit -> resubmit

File: study-builder/studies/basil/i18n/en.conf [hunk 2]
```diff
@@ -37,7 +37,8 @@
     "summary_created": "Completing the Loved One Survey will tell us about your loved one's experiences eating Basil",
     "summary_in_progress": "Submitting the Loved One Survey will tell us about your loved one's experiences eating Basil",
     "complete": "All set - thank you for letting us know about your loved one's basil eating experience",
-    "summary_error_present" : "Error Present. Please fix errors and resumit"
+    "summary_error_present" : "Error Present. Please fix errors and resumit",
```
Comment (Trimax) [inline study-builder/studies/basil/i18n/en.conf]: resumit -> resubmit

File: study-builder/studies/basil/study-activities.conf [hunk 3]
```diff
@@ -20,7 +20,29 @@
...
+          "precondition": ${_pex.is_comments_answered},
+          "expression": ${_pex.is_comments_error},
+          "messageTemplate": {
+            "templateType": "HTML",
+            "templateText": "$error_present_comm",
+            "variables": [
+              {
+                "name": "error_present_comm",
+                "translations": [
+                  { "language": "en", "text": ${i18n.en.aboutLovedOne.summary_error_present} },
```
Comment (Trimax) [inline study-builder/studies/basil/study-activities.conf:38]: Shouldn't we use new translations mechanism?

Comment (ssettipalli) [reply study-builder/studies/basil/study-activities.conf:38]: fon is not using new translations, so doesn't matter

**Review by Rinchan20 — APPROVED** [review]

**Pull request merged**